eureka/registry_test.go

```diff
@@ -124,6 +150,7 @@ var _ = Describe("Deregister", func() {
 							    "instance":[
 							       {
 								  "app":"APP-1",
+				                  "instanceId":"instance-1",
```

Comment:
Minor thing: looks like this line has a mixture of tabs and spaces, hence the slight formatting issue.

Reply:
@georgeharley Thanks. I _think_ I've fixed this. Thank goodness for IntelliJ's `Edit->Convert Indents->To Tabs`.